src/main/java/redis/clients/jedis/CommandObjects.java

```diff
@@ -2455,6 +2455,14 @@ protected CommandArguments commandArguments(ProtocolCommand command) {
         BuilderFactory.ENCODED_OBJECT);
   }
 
+  public final CommandObject<Object> evalReadonly(String script, List<String> keys, List<String> args) {
```

Comment:
Why not set keys to `String[]`

Reply:
Was trying to be consistent with the `eval` functions